frb/associate/frbassociate.py

```diff
@@ -14,8 +14,8 @@
 from astropy import wcs as astropy_wcs
 
 from frb import frb
-from frb.associate import bayesian
-from frb.associate import chance
+import bayesian
```

Comment:
not sure this will work for others.
Please put it back the way it was.  Thanks!

Reply:
Changed it back!